core/rawdb/accessors_chain_test.go

```diff
@@ -663,6 +745,38 @@ func makeTestReceipts(n int, nPerBlock int) []types.Receipts {
 	return allReceipts
 }
 
+func makeBlkBlob(n, nPerTx int) types.BlobTxSidecars {
+	if n <= 0 {
+		return nil
+	}
+	ret := make(types.BlobTxSidecars, n)
+	for i := 0; i < n; i++ {
+		blobs := make([]kzg4844.Blob, nPerTx)
+		commitments := make([]kzg4844.Commitment, nPerTx)
+		proofs := make([]kzg4844.Proof, nPerTx)
+		for i := 0; i < nPerTx; i++ {
+			io.ReadFull(rand2.Reader, blobs[i][:])
+			io.ReadFull(rand2.Reader, commitments[i][:])
```

Comment:
Shouldn't commitment be generated from blob instead of being random? Or am I missing something?